i18n/en/docusaurus-plugin-content-docs/current/get-started/tutorial.mdx

```diff
@@ -0,0 +1,283 @@
+
+Usually, these two layers are the best places to define your custom segments. Don’t be afraid to create new segments, they don’t cost you anything. For example, your Shared layer could consist of the following segments:
+
+- `ui` — the UI kit
+- `api` — the configuration of the API client
+- `i18n` — locales and internationalization providers
+- `store`\* — store providers
+- `routes` — route constants
+- `env` — environment variables
+
+\* not `model`, because it’s not a data model of Shared
+
+As you can imagine, most of these segments will be pretty small, and that’s okay. Their main goal is to help you find code quicker, not to pack as many things as possible.
+
+The App layer is usually the initialization code of your application. It’s where you define your routes, lay out your providers, connect styles. Sometimes your framework tells you where you should define your routes and configure things (for example, file-based routing like Next.js), in these cases it’s totally fine if the code on your App layer is not necessarily in the `app/` folder. The App layer is the last layer, thus it is queen! It can take as much space in the project source code as it needs.
+
+### Talking to the backend
+
+Now that most of our application is laid out, we need to fill it with some data. Our backend provides, but how do we retrieve and store it?
+
+We already touched on the API and Model segments in slices. They exist for this exact purpose, and they make the most sense on the Entities layer, since that’s where you need to fetch *and* store data. The API segment is also commonly found in Features, usually containing mutations, that is, operations that change the data on the backend.
+
+With some tech stacks you might not even need the Model segment for storage. For example, [TanStack Query](https://tanstack.com/query/latest) stores the server-side data in cache, allowing you to request a fetch and retrieve the data at the same time. Other stacks may have these operations separate, such as [Redux](https://redux.js.org/), where you dispatch an action to fetch and then use a selector to extract the fetched data from the store. In this case, the fetching code goes in API and the selector/reducer go to Model.
+
+### Recap
+
+We’ve covered a lot of ground here. Let’s sum it all up. Here’s the app we’re building, if you need a refresher:
+
+![Notes application screens](/img/tutorial/overview-light.jpg#light-mode-only)
+![Notes application screens](/img/tutorial/overview-dark.jpg#dark-mode-only)
+
+Our application’s code will be organized in layers, so let’s recap what each of them contains:
+
+- Shared
+    - UI segment — the UI kit of our application (we should be able to swap this for Ant or MUI if we want to)
+- Entities
+    - Note slice
+        - UI segment — the appearance of a note card
+        - API segment — query for a list of notes from the backend
+        - Model segment — storage of the fetched notes for app-wide access
```

Comment:
Probably should contain entity user/session so it would be clear from where we read user name `Sign-out (Lev)`